gen/build/src/lib.rs

```diff
@@ -298,7 +302,61 @@ fn generate_bridge(prj: &Project, build: &mut Build, rust_source_file: &Path) ->
     Ok(())
 }
 
+// Ensures that the argument of the include!() macro is either starting with the crate name, or an absolute path
```

Comment:
I think this check is somewhat too strict. For example the caller's build script is free to make any other directories available here and that wouldn't be a problem. Using https://crates.io/crates/pkg-config this could be a common pattern:

```rust
let python3 = pkg_config::probe_library("python3")?;
cxx_build::bridge("src/bridge.rs")
    .includes(python3.include_paths)
    .compile("repro");
```

```rust
extern "C" {
    include("Python.h");
    ...
}
```

Reply:
Under that circumstance, would it even be possible to check extra include directories inside cxx?  
We give the `cc::Build` object to the user and effectively lose control of how it's further configured.